inspirehep/modules/records/serializers/fields_export.py

```diff
@@ -0,0 +1,376 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import re
+
+from babel import Locale
+from isbn import ISBNError
+from six import text_type
+
+from idutils import is_arxiv_post_2007, normalize_isbn
+
+from inspire_utils.date import PartialDate
+from inspire_utils.record import get_value
+from inspirehep.modules.hal.utils import (
+    get_page_artid_for_publication_info,
+    get_conference_record,
+    get_conference_title,
+)
+
+from .config import COMMON_FIELDS_FOR_ENTRIES, FIELDS_FOR_ENTRY_TYPE
```

Comment:
Importing from the configuration means that the only way of overriding is modifying the code, which is highly undesiderable. A better way is to make this configuration part of the global configuration, and then request che configuration from the `current_app`.